### PR TITLE
feat(npm): Run npm install at end of generation process

### LINKED ISSUE #3: Run `npm install` at end of generation process
Are you against that ?

### PR DESCRIPTION
**Pull request by zckrs**
fix #3

### CODE REVIEW

**Comment by arthurvr** [comment]
@zckrs Tests fail. Can you take a look?

**Comment by zckrs** [comment]
Oups.

Fixed with `{ skipInstall: true }` in unit test.

File: test.js [hunk 1]
```diff
@@ -13,7 +13,9 @@ describe('generator', function () {
 				return;
 			}
 
-			this.generator = helpers.createGenerator('electron:app', deps);
+			this.generator = helpers.createGenerator('electron:app', deps, null, {
+				skipInstall: true
```
Comment (kevva) [inline test.js]: Inline it. `{skipInstall: true}`.

Comment (zckrs) [inline test.js]: Done. ^^'

**Comment by sindresorhus** [comment]
Wrong install method. Not the one suggested in the issue. This one doesn't tell the user what command to run manually.

https://github.com/sindresorhus/generator-electron/commit/79d084582c54028978815c70695bebf10a039138

**Comment by zckrs** [comment]
Right.

**Pull request merged**